include/invert_quda.h

```diff
@@ -569,7 +569,7 @@ namespace quda {
     /**
        @return Whether the solver is only for Hermitian systems
      */
-    virtual bool hermitian() = 0;
+    virtual bool hermitian() const noexcept = 0;
```

Comment:
Curious: why do we need this to be `noexcept`?

Reply:
Seemed like a best-practice type thing, there's no reason for it to not be (I guess), but I can certainly revert it.

Reply:
I see no reason to use `noexcept` here.  The only reason I've really seen where it matters is with respect to `move` semantics (e.g., ensuring move constructor is used over copy constructor) which obviously doesn't apply here.  I would prefer this can be reverted unless there's reason not to.

Reply:
Sounds good, I can get rid of it.

Reply:
https://github.com/lattice/quda/pull/1390/commits/9d1e63f9abde0ee62284db1fe1b156e2d902acf9